core/src/main/java/greencity/dto/advice/AdviceDTO.java

```diff
@@ -4,14 +4,14 @@
 import greencity.dto.user.HabitDictionaryDto;
 import javax.validation.constraints.Min;
 import javax.validation.constraints.NotBlank;
-import lombok.AllArgsConstructor;
-import lombok.Data;
-import lombok.NoArgsConstructor;
+import lombok.*;
```

Comment:
Better avoid this (*). You can configure Intellij IDEA not to group imports together

Reply:
fixed, made necessary configurations Intellij IDEA